packages/admin/admin/src/tabs/TabScrollButton.tsx

```diff
@@ -1,51 +1,58 @@
 import { ChevronLeft, ChevronRight } from "@comet/admin-icons";
-import {
-    ButtonBase,
-    ComponentsOverrides,
-    TabScrollButtonClassKey as MuiTabScrollButtonClassKey,
-    TabScrollButtonProps as MuiTabScrollButtonProps,
-    Theme,
-} from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ButtonBase, ButtonBaseProps, ComponentsOverrides } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
 import isMobile from "is-mobile";
 import * as React from "react";
 
-export type TabScrollButtonClassKey = MuiTabScrollButtonClassKey;
+export type TabScrollButtonClassKey = "root" | "vertical" | "disabled";
```

Comment:
I think you could remove `disabled` from the `ClassKey`, as it is not being used in the `overridesResolver` and any Styling could be applied with `:disabled` in the Root-Slot. 

This would be an incompatible change however, since `disabled` was included in  the previously used  `MuiTabScrollButtonClassKey` type. 
Then you'd have to add a changeset, maybe similar to: https://github.com/vivid-planet/comet/blob/4f8ad320e584b9b27b89e1af9d90523d7a78e1b5/.changeset/ten-turkeys-poke.md?plain=1#L29